lib/buildFile.js

```diff
@@ -33,6 +34,8 @@ function buildFile(destination, format, platform, dictionary, filter) {
   if (!destination || typeof destination !== 'string')
     throw new Error('Please enter a valid destination');
 
+  var dest = destination;
```

Comment:
probably I would not do this, but create a variable `fullDestination` (or something similar, I am not sure about the name) which is the one used and manipulated, and leave the value `destination` untouched, so it can be used directly in the construction of the title if propertyNamesCollisionCount > 0

Reply:
agreed